korma/src/commonMain/kotlin/com/soywiz/korma/geom/vector/Edge.kt

```diff
@@ -50,18 +50,7 @@ class Edge {
             val Cy: Double = b.ay.toDouble()
             val Dx: Double = b.bx.toDouble()
             val Dy: Double = b.by.toDouble()
-            val a1 = By - Ay
-            val b1 = Ax - Bx
-            val c1 = a1 * (Ax) + b1 * (Ay)
-            val a2 = Dy - Cy
-            val b2 = Cx - Dx
-            val c2 = a2 * (Cx) + b2 * (Cy)
-            val determinant = a1 * b2 - a2 * b1
-            if (determinant == 0.0) return false
-            val x = (b2 * c1 - b1 * c2) / determinant
-            val y = (a1 * c2 - a2 * c1) / determinant
-            out(floorCeil(x), floorCeil(y))
-            return true
+            return getIntersectXY(Ax, Ay, Bx, By, Cx, Cy, Dx, Dy, out)
```

Comment:
Notice the `floorCeil`. To work the same, it should be:
```
return getIntersectXY(Ax, Ay, Bx, By, Cx, Cy, Dx, Dy) { x, y -> out(floorCeil(x), floorCeil(y)) }
```

Also. We are working on the korge-next repository https://github.com/korlibs/korge-next . Can you make the PR there instead? Thanks!

Reply:
Of corse :)